corehq/apps/reports/display.py

```diff
@@ -89,7 +89,7 @@ def metadata(self):
         try:
             return FormType.forms_by_xmlns(self.domain, self.xmlns, self.app_id)
         except Exception, e:
-            _assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'))
+            _assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'), notify_admins=True)
```

Comment:
mmm, not sure, i guess i was on autopilot haha

On Fri, Oct 9, 2015 at 12:38 AM, Cory Zue <notifications@github.com> wrote:

> In corehq/apps/reports/display.py
> <https://github.com/dimagi/commcare-hq/pull/8559#discussion_r41596873>:
>
> > @@ -89,7 +89,7 @@ def metadata(self):
> >          try:
> >              return FormType.forms_by_xmlns(self.domain, self.xmlns, self.app_id)
> >          except Exception, e:
> > -            _assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'))
> > +            _assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'), notify_admins=True)
>
> wasn't blocking but curious why you're still notifying yourself too
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8559/files#r41596873>.
>